interfaces/uss_qualifier/uss_qualifier.yaml

```diff
@@ -141,14 +134,15 @@ components:
                         Tests result if task has completed.
                 generated_flight_records:
                     type: array
-                    items: string
+                    items: FlightRecordID
```

Comment:
`items: FlightRecordID` is required to be replaced with:
`items:
   $ref: "#/components/schemas/FlightRecordID"`

Reply:
Thanks, will fix